Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1188,17 +1219,17 @@ private static void runQuickTests(MinioClient minioClient)
                         await minio.RemoveObjectAsync(bucketName, objectName);
                     }
             }
-            catch (Exception e)
+            catch (Exception ex)
             {
-                new MintLogger("GetObject_Test3","Tests whether GetObject returns all the data",TestStatus.FAIL,(DateTime.Now - startTime),"",e.Message, e.ToString()).Log();
-                Assert.Fail();
+                    Assert.Fail();
+                    throw ex;
             }
             await TearDown(minio, bucketName);
-            new MintLogger("GetObject_Test3","Tests whether GetObject returns all the data",TestStatus.PASS,(DateTime.Now - startTime)).Log();
+            new MintLogger("GetObject_Test3",getObjectSignature2,"Tests whether GetObject returns all the data",TestStatus.PASS,(DateTime.Now - startTime)).Log();
```

Comment:
function field should have the function definition itself.

Reply:
getObjectSignature2 is just the function signature. Not sure what else is expected here?